### PR TITLE
Added repos to translate some of the codified fields.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: exam-processor/src/test/resources/SBAC-IAB-FIXED-G4M-G-MATH-4.xml [hunk 1]
```diff
@@ -31,8 +31,8 @@
         <ExamineeAttribute name="PrimaryDisabilityType" value="" context="INITIAL" contextDate="2016-10-12T21:51:46.003"/>
         <ExamineeAttribute name="Birthdate" value="1900-01-01" context="FINAL" contextDate="2016-10-12T22:00:49.069"/>
         <ExamineeAttribute name="FirstName" value="Name116" context="FINAL" contextDate="2016-10-12T22:00:49.069"/>
-        <ExamineeAttribute name="Sex" value="F" context="FINAL" contextDate="2016-10-12T22:00:49.069"/>
-        <ExamineeAttribute name="GradeLevelWhenAssessed" value="FOURTHGRADE" context="FINAL" contextDate="2016-10-12T22:00:49.069"/>
+        <ExamineeAttribute name="Sex" value="Female" context="FINAL" contextDate="2016-10-12T22:00:49.069"/>
+        <ExamineeAttribute name="GradeLevelWhenAssessed" value="04" context="FINAL" contextDate="2016-10-12T22:00:49.069"/>
         <ExamineeAttribute name="LastOrSurname" value="LastName116" context="FINAL" contextDate="2016-10-12T22:00:49.069"/>
```
Comment (agorina) [inline exam-processor/src/test/resources/SBAC-IAB-FIXED-G4M-G-MATH-4.xml]: @malaffoon , just wanted to point out that I had to make this changes in order for it to work with the TRT spec. I am not sure if you have 'sex' on your list of questions. I added this to the logical data model page.

Comment (malaffoon) [inline exam-processor/src/test/resources/SBAC-IAB-FIXED-G4M-G-MATH-4.xml]: Yes, i called out the allowed values for Sex in my notes.

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultTDSReportProcessor.java [hunk 2]
```diff
@@ -37,15 +37,20 @@ public DefaultTDSReportProcessor(final ExamineeParser examineeParser,
     @Override
     public void process(final TDSReport report, final TenancyChain tenancyChain) {
         final ParsedExaminee parsedExaminee = examineeParser.parse(report.getExaminee());
-        final School school = parsedExaminee.getSchool();
-        //TODO: authorize(); need to had TenancyChain methods to support this,
-        //TODO: tenancyChain.hasRoleForState("ASMTDATALOAD", "AP") won't be good enough
+
+        if (!authorize(tenancyChain, parsedExaminee.getSchool())) return;
```
Comment (malaffoon) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultTDSReportProcessor.java]: This pattern may lead to setting import status in a bunch of places. Perhaps you could have the assumption of the happy path with exceptions indicating failure and giving the status code and message. Something like:
```
  try {
    process();
    updateImportStatus(PROCESSED, null);
  } catch (ImportException e) {
    updateImportStatus(e.getStatus(), e.getMessage());
  }
```

Comment (agorina) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultTDSReportProcessor.java]: Ok.

**Pull request merged**